### ISSUE TITLE
Trying the action helper with bazel 4.0.0 results in a failure

### ISSUE THREAD

**Issue by codergs90** [jwlawson/actions-setup-bazel#11]
**Summary:**

Tried below setting in the actions yml file

```
 # Install bazel using a helper action
      - name: Setup bazel
        uses: jwlawson/actions-setup-bazel@v1.5
        with:
          bazel-version: '4.0.0'
      # Runs a single command to ensure bazel 4.0.0 was installed
      - name: Use bazel
        run: bazel --version
```

Getting below error:
```
Run jwlawson/actions-setup-bazel@v1.5
  with:
    bazel-version: 4.0.0
    github-api-token: ***
Error: Found more than one matching asset for bazel version 4.0.0
```

**Expectation:**
The action should work with bazel 4.0.0

**Comment by jwlawson** [comment]
Thanks for bringing this up, I'll take a look. My first guess is that it finds both the `nojdk` and normal assets, so the action should have a way to filter them.

**Comment by jwlawson** [comment]
This should be fixed by #12. Can you try with version 1.6 and see if it works?
